scripts/extraction/merge_to_MEDS_cohort.py

```diff
@@ -32,19 +37,19 @@ def read_fn(sp_dir: Path, unique_by: list[str] | str | None) -> pl.LazyFrame:
         case None:
             pass
         case "*":
-            df = df.unique(maintain_order=False)
+            df = df.unique(maintain_order=True)
         case list() if len(unique_by) == 0 and all(isinstance(u, str) for u in unique_by):
             subset = []
             for u in unique_by:
                 if u in df.columns:
                     subset.append(u)
                 else:
                     logger.warning(f"Column {u} not found in dataframe. Omitting from unique-by subset.")
-            df = df.unique(maintain_order=False, subset=subset)
+            df = df.unique(maintain_order=True, subset=subset)
         case _:
             raise ValueError(f"Invalid unique_by value: {unique_by}")
 
-    return df.sort(by=["patient_id", "timestamp"], multithreaded=False)
+    return df.sort(by=["patient_id", "timestamp"], maintain_order=True)
```

Comment:
Keep the `multithreaded=False` please or make it configurable via the stage config. It can get expensive for datasets with lots of measurements, and eICU in particular was giving me trouble.